<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anfitrion</class>
 <widget class="QWidget" name="Anfitrion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #C0C4CF;
border-right: 1px solid;

}
</string>
   </property>
   <widget class="QWidget" name="Menu">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Img.qrc">:/img/Iconos/logosinfondo1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="salida">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/salida.png</normaloff>:/img/Iconos/salida.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Disponibilidad_Mesas">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
     </property>
     <property name="text">
      <string>Disponibilidad</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/reserva-de-mesa.png</normaloff>:/img/Iconos/reserva-de-mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>0</y>
     <width>591</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #DFE3E5;
border-right: 1px solid;

}</string>
   </property>
   <widget class="QWidget" name="Inicio_Anfitrion">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>411</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Img.qrc">:/img/Iconos/platillos-png.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>0</y>
       <width>571</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SuckerPunch</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#8E5329;</string>
     </property>
     <property name="text">
      <string>Bienvenido Anfitri√≥n</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Mesa_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>CreativeBlock BB</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/mesa.png</normaloff>:/img/Iconos/mesa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>augie</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string>NO DISPONIBLE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>augie</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #008000;</string>
     </property>
     <property name="text">
      <string>DISPONIBLE</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Mesas_Disponibles"/>
  </widget>
 </widget>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
